src/assets/scss/style.scss

```diff
@@ -18,3 +18,40 @@ body {
     width: 100%;
     height: 100%;
 }
+
+.text-input-box{
+    width: 600px;
+    margin: 20px;
+    margin-bottom: 20px;
+    background-color: $primary-light;
+    color: $other-light;
+}
+
+.text-input-box h3, p{
+    margin-left: 10px;
+    margin-top: 10px;
+}
+
+.input-box{
+    appearance: none;
+    border-radius: 25px;
+    border: 2px solid $accent-light;
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.185);
+    padding: 20px;
+    width: 80%; 
+    margin: 5px;
+    height: 15px;
+    transition: .3s box-shadow;
+    background-color: $primary-light;
+    color: $other-light;
+    outline: none;
+    margin-bottom: 20px;
+}
+.input-box::placeholder{
+    color: $other-dark;
+}
+.input-box:focus{
+    background-color: $primary-light;
+    border: 2px solid $accent-light;
+}
+
```

Comment:
All the code must be in a specific css file for this input

Reply:
done!